client/src/components/DeleteReview.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+
+const DeleteButton = (props) =>{
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const { reviewId, restaurantId } = props 
+
+  const deleteReview = async () => {
+    try {
+        const response = await fetch (`/api/v1/reviews/${reviewId}`, {
+        method: "delete",
+        headers: new Headers({
+          "Content-Type": "application/json"
+        })
+      })
+       if(!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      setShouldRedirect(true)
+      return { status: "ok" }
```

Comment:
we don't want to redirect because we are already on the show page - here we need to update state for all the reviews for this restaurant to remove the review from state that was just deleted